tests/performance/locust_settings.py

```diff
@@ -150,8 +151,19 @@ def _serialize_host(self, url: AnyHttpUrl, info: SerializationInfo) -> str:
 
 
 if __name__ == "__main__":
-    settings = LocustSettings()
-    env_vars = [
-        f"{key}={val}" for key, val in json.loads(settings.model_dump_json()).items()
-    ]
+    locust_settings = LocustSettings()
```

Comment:
TIP: pydantic settings can directly read from different sources, not only env-vars. One of them is CLI via `sys.argv` 
https://docs.pydantic.dev/latest/concepts/pydantic_settings/#command-line-support